app/controllers/assessor_interface/preliminary_checks_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module AssessorInterface
+  class PreliminaryChecksController < BaseController
+    before_action :authorize_assessor
+
+    def edit
+      @form = PreliminaryCheckForm.new(assessment: application_form.assessment)
+    end
+
+    def update
+      @form =
+        PreliminaryCheckForm.new(
+          form_params.merge(assessment: application_form.assessment),
+        )
+
+      if @form.save
+        CreatePreliminaryCheckNote.call(
+          application_form:,
+          author: current_staff,
+          preliminary_check_complete: @form.preliminary_check_complete,
+        )
+
+        redirect_to assessor_interface_application_form_path(application_form)
+      else
+        render :edit
```

Comment:
```suggestion
        render :edit, status: :unprocessable_entity
```

I don't think it matters much but it ensures we use the right HTTP codes.